modules/va_forms/app/controllers/va_forms/v0/forms_controller.rb

```diff
@@ -21,7 +21,8 @@ def show
 
       def get_forms
         if params[:query].present?
-          Form.where('form_name ilike ? OR title ilike ?', "%#{params[:query]}%", "%#{params[:query]}%")
+          query = params[:query].strip
+          Form.where('form_name ilike ? OR title ilike ?', "%#{query}%", "%#{query}%")
```

Comment:
Could you add a test in https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/va_forms/spec/requests/v0/forms_request_spec.rb to show there's match when a trailing space is added?  (Sometimes I'll comment out my change to be sure the test fails without my change if I didn't test first.)